### Performing Arithmetic Operations Using Python

In [1]:
2+3+9

14

In [2]:
99-73

26

In [3]:
23.54 * -1432

-33709.28

In [4]:
100/7 # Complete quotient with decimal too

14.285714285714286

In [5]:
100// 7 # Only the quotient as an Integer

14

In [6]:
100 % 7 # Displays the remainder

2

In [7]:
5 **3 # raise to the power

125

As you might expect, operators like `/` and `*` take precedence over other operators like `+` and `-` as per mathematical conventions. You can use parentheses, i.e. `(` and `)`, to specify the order in which operations are performed.

In [8]:
((2 + 5) * (17 - 3)) / (4**3)

1.53125

Python supports the following arithmetic operators:

| Operator   | Purpose           | Example     | Result    |
|------------|-------------------|-------------|-----------|
| `+`        | Addition          | `2 + 3`     | `5`       |
| `-`        | Subtraction       | `3 - 2`     | `1`       |
| `*`        | Multiplication    | `8 * 12`    | `96`      |
| `/`        | Division          | `100 / 7`   | `14.28..` |
| `//`       | Floor Division    | `100 // 7`  | `14`      |    
| `%`        | Modulus/Remainder | `100 % 7`   | `2`       |
| `**`       | Exponent          | `5 ** 3`    | `125`     |


Try solving some simple problems from this page:
https://www.math-only-math.com/worksheet-on-word-problems-on-four-operations.html . 

In [3]:
# 1.The population of a town is 198568. Out of them 45312 are men and 35678 are women. Find the number of children
# in the town.
(198568 - (45312 + 35678))

117578

In [4]:
# 2. A shopkeeper has 2425 boxes of 24 pencils each. How many pencils do all the boxes have in all?
(24 * 2425)

58200

In [5]:
# 3. Linda bought a coat for $2265 and a saree for $2150. She gave $5000 to the shopkeeper. 
# How much money did the shopkeeper return to her?
(5000 - (2265 + 2150))

585

In [8]:
# 13. Maria bought 96 toys priced equally for $12960. The amount of $1015 is still left with her. Find the cost of 
# each toy and the amount she had.
print(12960/96)
print(12960 + 1015)

135.0
13975


## Solving multi-step problems using variables

Let's try solving the following word problem using Python: 

> A grocery store sells a bag of ice for $1.25 and makes a 20% profit. If it sells 500 bags of ice, how much total profit does it make?

We can list out the information provided and gradually convert the word problem into a mathematical expression that can be evaluated using Python. 

*Cost of ice bag ($)* = 1.25

*Profit margin* = 20% = .2

*Profit per bag ($)* = profit margin * cost of ice bag = .2 * 1.25

*No. of bags* = 500

*Total profit* = no. of bags * profit per bag = 500 * (.2 * 1.25)

In [9]:
(500 * (0.2*1.25))

125.0

The variables `cost_of_ice_bag`, `profit_margin`, and `number_of_bags` now contain the information provided in the word problem. We can check the value of a variable by typing its name into a cell. We can combine variables using arithmetic operations to create other variables.

In [10]:
cost_of_ice_bag = 1.25

In [11]:
profit_margin = 0.2

In [12]:
number_of_bags = 500

In [13]:
profit_margin

0.2

In [14]:
profit_per_bag = (profit_margin * cost_of_ice_bag)

In [15]:
profit_per_bag

0.25

In [16]:
total_profit = profit_per_bag * number_of_bags

In [17]:
total_profit

125.0

In [18]:
print("The grocery store makes a total profit of ${}".format(total_profit))

The grocery store makes a total profit of $125.0
